samples/bluetooth/nrf_dm/README.rst

```diff
@@ -0,0 +1,188 @@
+.. _ble_nrf_dm:
+
+Bluetooth: nRF Distance Measurement with Bluetooth LE discovery
+###############################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF Distance Measurement sample demonstrates the functionality of the Distance Measurement (DM) subsystem.
+It shows how to use DM to measure the distance between devices.
+The Bluetooth Heart Rate Service is running simultaneously.
+
```

Comment:
Move the Requirements section here to align with the sample template: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/sample_README.html